pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -261,6 +261,12 @@ func (els *EventListenerStatus) SetConditionsForDynamicObjects(conditions v1beta
 			Message: cond.Message,
 		})
 	}
+
+	els.SetCondition(&apis.Condition{
```

Comment:
question: why do we need this in this PR?

Reply:
Added as part of this PR because EL for Knative when succeed was not displaying any messages so added this change as part of this PR itself